docs/known_issues.md

```diff
@@ -14,15 +14,24 @@ As a workaround, use `tag.turbo_frame` instead of `turbo_frame_tag`.
 
 Note: For the same functionality as `turbo_frame_tag(my_model)`, use `tag.turbo_frame(id: dom_id(my_model))`.
 
-## form_for compatibility
+## Compatibility with Rails form helpers
 
 ViewComponent [isn't compatible](https://github.com/viewcomponent/view_component/issues/241) with `form_for` helpers by default.
 
-## Inconsistent controller rendering behavior between Rails versions
+Passing a form object reference (usually `f`) to a view component works for simple cases, like `f.text_field :name`.
+Content will be ill-ordered or duplicated in complex cases (such as passing blocks to form helpers or when nesting components).
 
-In versions of Rails < 6.1, rendering a ViewComponent from a controller doesn't include the layout.
+Consider following options:
 
-## Forms don't use the default form builder
+- Render a form end-to-end within one view component
+- Render a [classical view partial](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) within the view component, which includes the form
+- Use `[FormBuilder](https://guides.rubyonrails.org/form_helpers.html#customizing-form-builders)` to create reusable form components.
+  Note: There are multiple approaches how to implement a FormBuilder:
+  - Classic approach (without ViewComponent)
+  - Using ViewComponent (for example, [view_component-form gem](https://github.com/pantographe/view_component-form))
```

Comment:
I may be contradicting [my own comment](https://github.com/ViewComponent/view_component/pull/1644#discussion_r1092464568) here, but I now think we need to describe the overall approach.

```suggestion
  - Using a FormBuilder that overrides all field helpers to render a ViewComponent, so that you can customize each field individually (for example, [view_component-form gem](https://github.com/pantographe/view_component-form))
```